plans/smoke.fmf

```diff
@@ -0,0 +1,4 @@
+summary:
+  Basic smoke test.
```

Comment:
Nitpick: I'd suggest not using full stops at the end of summaries. I personally view it similarly to commit messages and the first lines but it's more of a personal taste :)